From 25d8ad09d86590d2731ebfa69595f071e44ccbfd Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 8 Nov 2017 14:32:20 +0800
Subject: [PATCH 32/35] bluetooth: btwilink: remove MODULE_DEVICE_TABLE to
 avoid be installed at boot time

btwilink should be actived after kim from apps, so remove the module table
of definition.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
(cherry picked from commit afbc2f80f3f6ffc5b047d1383536a2b0272114fd)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/bluetooth/btwilink.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/bluetooth/btwilink.c b/drivers/bluetooth/btwilink.c
index e6db946..b7afbdd 100644
--- a/drivers/bluetooth/btwilink.c
+++ b/drivers/bluetooth/btwilink.c
@@ -282,7 +282,6 @@ static int ti_st_send_frame(struct hci_dev *hdev, struct sk_buff *skb)
 	},
 	{}
 };
-MODULE_DEVICE_TABLE(of, btwilink_of_match);
 
 static int bt_ti_probe(struct platform_device *pdev)
 {
-- 
1.7.9.5

